src/CSSFramework/BootstrapCSSFramework.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace TheWebmen\ElementalGrid\CSSFramework;
+
+use DNADesign\Elemental\Models\BaseElement;
+
+final class BootstrapCSSFramework implements CSSFrameworkInterface
+{
+    /**
+     * @var BaseElement
+     */
+    private $baseElement;
+
+    private const COLUMN_CLASSNAME = 'col';
+
+    private const ROW_CLASSNAME = 'row';
+
+    private const CONTAINER_CLASSNAME = 'container';
+
+    private const FLUID_CONTAINER_CLASSNAME = 'container-fluid';
+
+    /**
+     * @param BaseElement $baseElement
+     */
+    public function __construct($baseElement)
```

Comment:
Missing type-hint

Reply:
And missing this on other places too, don't know if you want to tackle this or for later.